src/components/EditableTypography/EditableTypography.module.scss

```diff
@@ -0,0 +1,34 @@
+@import "~monday-ui-style/dist/mixins";
```

Comment:
All `EditableTypography` files are previously the new `EditableHeading` component that became more generic